health/health.d/anomalies.conf

```diff
@@ -5,7 +5,7 @@ template: anomalies_anomaly_probabilities
   lookup: average -2m foreach *
    every: 1m
     warn: $this > 50
-    info: average anomaly probability > 50% for last 2 minutes
+    info: average anomaly probability over the last 2 minutes
```

Comment:
Probably distribution have different mean/averag, I think to hide the 50% can conduct to different conclusions.  @vlvkobal and @andrewm4894 am I missing something?

Reply:
If the alarm is up there is value

`WARN | average anomaly probability over the last 2 minutes| 73`